RELEASING.md

```diff
@@ -3,3 +3,4 @@
 1. Create a `vX.Y.Z` tag and release notes at https://github.com/lightbend/akkaserverless-java-sdk/releases
 2. CircleCI will automatically publish to Sonatype OSSRH (Maven Central) based on the tag.
 3. Update the `akkaserverless-sdk.version` in the `samples/*/pom.xml` files to the released version.
+4. Update the [docs Makefile](https://github.com/lightbend/akkaserverless-java-sdk/blob/main/docs/Makefile#L38) to the released version.
```

Comment:
I think this should be step 1 (before releasing), because `docs/current` is updated by the release bot when creating the tag

or did I misunderstand something?

Reply:
Ah! I didn't know that. I'll change it